pkg/charts/charts.go

```diff
@@ -13,7 +13,7 @@ import (
 	"k8s.io/helm/pkg/timeconv"
 )
 
-const versionPlaceholder = "{version}"
+const versionPlaceholder = "linkerdVersionValue"
```

Comment:
I suspect that this was missed in https://github.com/linkerd/linkerd2/pull/4373

Reply:
good catch :+1: